config/config-api/src/com/thoughtworks/go/util/XmlUtils.java

```diff
@@ -95,4 +98,16 @@ public static boolean doesNotMatchUsingXsdRegex(Pattern pattern, String textToMa
     public static boolean matchUsingRegex(Pattern pattern, String textToMatch) {
         return pattern.matcher(textToMatch).matches();
     }
+
+    public static String stripProlog(String line) {
+        if (line.contains(PROLOG_START)) {
```

Comment:
@ketan - great point. I added this to the test cases and implemented your regex suggestions. Thanks for your patience!